Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1270,6 +1270,16 @@ final class StatusTableViewController: LoopChartsTableViewController {
         } else {
             let vc = UIAlertController(premealDurationSelectionHandler: { duration in
                 let startDate = Date()
+                
+                guard self.workoutMode == false else {
```

Comment:
What happens if `workoutMode` is `nil`? Does this need to be `self.workoutMode != true`?